### DISCUSSION TITLE
Slack invite?

### DISCUSSION THREAD

**Discussion by joebowbeer** [Infisical/infisical#1523] [Q&A] [answered]
The slack [link](https://infisical.com/slack) is stale, apparently.

Is the slack channel still happening?

If so, how to join?

**Comment by maidul98** [comment, marked as answer]
Looks like it works on my end. What error are you getting? Can you please share a screenshot

**Comment by joebowbeer** [comment]
Works now. Closing

**Reply by Daemoen** [reply]
It seems to only allow infisical.com domain users.  I thought the slack channel was for the community at large?

**Reply by maidul98** [reply]
Please use this link: https://infisical-users.slack.com/

**Reply by Daemoen** [reply]
Unfortunately, it doesn't work.  When you attempt to register (or login), it requires an infisical.com domain, and asks you to contact the WS admins for assistance.
![Screenshot 2024-07-02 at 1 26 03 PM](https://github.com/Infisical/infisical/assets/1039884/89d2bef1-d125-447a-a6ea-d4f66b45d90a)

**Reply by maidul98** [reply]
I tried this link and it works on my end without a infisical email: https://infisical.com/slack

**Reply by Daemoen** [reply]
Finally worked.  Not sure why it hasnt over the past 24 hours.  Thanks @maidul98 :)

**Reply by ubermensch19** [reply]
Hey its not working for me. How to join the slack or its only for internal members? It shows to have an <>@infisical.com email address.

@maidul98
